README.md

```diff
@@ -1,6 +1,14 @@
+_Brought to you by the Bevy Jam working group._
```

Comment:
This is a bit better. TBH we could move this to GitHub's `About` section instead?

Reply:
I prefer that one to say "This template is a great way to get started on a new [Bevy](https://bevyengine.org/) game — especially for a game jam!"

Reply:
(Added that to the GitHub description just now.)

Reply:
Well, I'm unsure, but this is definitely fine for now.